media/css/firefox/family/components/_agreement.scss

```
@@ -35,12 +35,20 @@
     }
 }
 
-// todo: interaction styles
 .c-download {
+    @include f3.border(transparent);
     border-radius: $border-radius-sm;
-    font-weight: bold;
     background-color: f3.$white;
     color: f3.$black;
-    @include f3.border(transparent);
+    font-weight: bold;
     padding: $spacing-sm $spacing-2xl;
+    text-decoration: none;
+    transition: background-color 100ms color 100ms;
```

I'm seeing an invalid property value error in inspector tools, maybe we need a comma?
```suggestion
    transition: background-color 100ms, color 100ms;
```